### PR TITLE
SEO-172815-UWP-controls

### PR DESCRIPTION
**Pull request by Edithabich**
Hi @MallikaRK ,
Have added the contents, please review.
Thanks.

Regards,
Edith.

### CODE REVIEW

File: uwp/AutoComplete/Getting-Started.md [hunk 1]
```diff
@@ -54,4 +54,6 @@ Dim textBox As New SfTextBoxExt()
 
 {% endtabs %}
 
+N> You can refer to our [UWP AutoComplete feature tour](https://help.syncfusion.com/uwp/autocomplete/getting-started) page to know about its other groundbreaking feature representations. You can also explore our [UWP AutoComplete example](https://apps.microsoft.com/store/detail/syncfusion-essential-studio-for-uwp/9NBLGGH5WNGV) to understand how to present and manipulate data.
```
Comment (MallikaRK) [inline uwp/AutoComplete/Getting-Started.md:57]: Hi @Edithabich , update feature tour page link. you have added UG page here. 
image

File: uwp/Numeric-TextBox/Getting-Started.md [hunk 2]
```diff
@@ -58,4 +58,6 @@ Dim numericTextBox As New SfNumericTextBox()
 
 ## See also
 
-[How to restrict the decimal values in SfNumericTextBox](https://www.syncfusion.com/kb/6957/how-to-restrict-the-decimal-values-in-sfnumerictextbox)
\ No newline at end of file
+[How to restrict the decimal values in SfNumericTextBox](https://www.syncfusion.com/kb/6957/how-to-restrict-the-decimal-values-in-sfnumerictextbox)
```
Comment (MallikaRK) [inline uwp/Numeric-TextBox/Getting-Started.md:61]: Update our latest KB link here.

**Review by MallikaRK — CHANGES_REQUESTED** [review]
Commented

File: uwp/Numeric-TextBox/Getting-Started.md (hunk 2 above)
Comment (Edithabich) [reply uwp/Numeric-TextBox/Getting-Started.md]: Link updated

**Review by MallikaRK — APPROVED** [review]
Approved

File: uwp/AutoComplete/Getting-Started.md [hunk 3]
```diff
@@ -54,4 +54,6 @@ Dim textBox As New SfTextBoxExt()
 
 {% endtabs %}
 
+N> You can refer to our [UWP AutoComplete feature tour](https://www.syncfusion.com/uwp-ui-controls/autosuggestbox) page to know about its other groundbreaking feature representations. You can also explore our [UWP AutoComplete example](https://apps.microsoft.com/store/detail/syncfusion-essential-studio-for-uwp/9NBLGGH5WNGV) to understand how to present and manipulate data.
```
Comment (Girija-Rajendran) [inline uwp/AutoComplete/Getting-Started.md:57]: [UWP AutoComplete](https://www.syncfusion.com/uwp-ui-controls/autosuggestbox) feature tour page..........

update latest content
image

Comment (Edithabich) [reply uwp/AutoComplete/Getting-Started.md:57]: resolved

**Review by Aishwarya-Ganesan — APPROVED** [review]
Changes are fine.

**Pull request merged**